gordo/reporters/mlflow.py

```diff
@@ -0,0 +1,440 @@
+from contextlib import contextmanager
```

Comment:
Is this whole file copied in, or are there any changes as well?

Reply:
ya.. I think it wrongly attributes this to a delete and 'new'. When actually it was moved _and_ renamed. No changes, aside from adding the `MlFlowReporter` class.